assets/terraform/gce/bootstrap/centos.sh

```diff
@@ -2,12 +2,28 @@
 #
 # VM bootstrap script for CentOS/RHEL
 #
-set -exuo pipefail
+set -o errexit
+set -o errtrace
+set -o xtrace
+set -o nounset
+set -o pipefail
+
+rm -f /var/lib/bootstrap_*
```

Comment:
Like I mentioned elsewhere, I'm reusing these scripts on test VMs that are being restarted multiple times so it would make sense there. But you're right - in context of robotest this is not strictly required.